.github/workflows/push.yml

```diff
@@ -0,0 +1,230 @@
+name: Pipeline
```

Comment:
Both push.yml and build.yml have the same name "Pipeline".
Would be good to have different names in case of debugging.

Reply:
Yes, we are not running both workflows at the same time

1. build workflow will run for PR
2. push workflow will run after the PR merge

But let me change it